### DISCUSSION TITLE
In the smart connection v2.3.40, I can't use my prompts  by '@'

### DISCUSSION THREAD

**Discussion by comrade1059** [brianpetro/obsidian-smart-connections#910] [Q&A]
After I send my prompt by '@', the chat model seems not to see it. But when I send other notes, it functions very good. Anyone could help me with that? I'd be more than appreciative!!

**Comment by comrade1059** [comment]
it just like this.
![屏幕截图 2024-12-05 194726](https://github.com/user-attachments/assets/27d0ee1c-7872-4ad6-8da5-cbe8fac0c30e)
i've tried many methods, but i can't solve it. sad for that.

**Comment by comrade1059** [comment]
I have resolved this issue by removing 'smart-prompts' from the folder exclusions in version 2.2.85, reloading the source, and restarting the plugin and Obsidian! I am so glad about it! However, I still do not understand if this means that if I exclude a file, I will be completely unable to reference it during my interactions with the chat model? Thank you once again!

**Reply by brianpetro** [reply]
You are correct that exclusions in the settings will prevent the chat from being able to read the file. 

I made a note to prevent excluded files from appearing in the context selection (`@`) UI. This should help reduce some confusion.

I also made a note to potentially add an extra setting for whether the Smart Chat should respect exclusion settings when files are specified directly. 

I'm also considering completely dropping the exclusion settings if the file is manually added to the chat. 

Thanks for following up and clarifying how you resolved the issue 🌴
